packages/client/src/components/DateRangePicker.tsx

```diff
@@ -89,6 +89,9 @@ function isPresetNavButton(button: IPresetButton): button is IPresetNavButton {
 
 interface IDateRangePickerProps extends WrappedComponentProps, IDateRange {
   onDatesChange: ({ startDate, endDate }: IDateRange) => void
+  hidePresetBar?: boolean
+  closeModalFromHOC?: () => void
+  usedInsideHOC?: boolean
```

Comment:
I think we should be able to use only a single boolean in this case as both of them are kinda doing the same thing

Reply:
Thanks @Zangetsu101. I have removed the hidePresetBar props